README.md

```diff
@@ -3,6 +3,15 @@
 
 [![logo](views/resources/logos/assembly.png)](https://godbolt.org/)
 
+
+<ul style="display: flex; flex-direction: row; flex-wrap: wrap; font-size: 1em;">
```

Comment:
Fly-by; does this need to be HTML? The rest of the document is markdown.

Reply:
I formatted it this way because I saw it in some popular repositories. I think it would be useful for navigation.

Let me know if you'd like me to elaborate on any specific formatting choices!

Reply:
GitHub already has a built-in table of contents function. I'm not sure if this is necessary?